### PR TITLE
Auto Bump Kubeaddons Kommander

### PR DESCRIPTION
**Pull request by orsenthil**
####  Autobump Kubeaddons Kommander

https://jira.d2iq.com/browse/D2IQ-74160

### CODE REVIEW

**Review by faiq — APPROVED** [review]

File: Dispatchfile [hunk 1]
```diff
@@ -1,31 +1,55 @@
 #!mesosphere/dispatch-starlark:v0.8
 # vi:syntax=python
-#
-#
 
 load("github.com/mesosphere/dispatch-catalog/starlark/stable/git@0.0.7", "git_resource")
-load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@0.0.7", "tag")
+load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@0.0.7", "tag", "pull_request", "push", "cron")
```
Comment (faiq) [inline Dispatchfile:5]: do we need cron?

Comment (orsenthil) [reply Dispatchfile]: No. Just removed it.

**Review by joejulian — APPROVED** [review]

**Pull request merged**